app/src/testDemo/resources/robolectric.properties

```diff
@@ -0,0 +1,17 @@
+#
+# Copyright 2025 The Android Open Source Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+sdk = 35
```

Comment:
Why not 36?

Reply:
Robolectric doesn't support 36 yet so it's throwing an error unless I specifically target 35. Support for 36 will be added in 4.16.
https://github.com/robolectric/robolectric/releases/tag/robolectric-4.15